Events/feedback_visatt.m

```diff
@@ -0,0 +1,228 @@
+out = true; %If out == false, the loading of the experiment will be cancled.
+end
+
+function out = enabled()
+out = true; %If this function returns false, it will not be included.
+end
+
+function out = getLoadFunction()
+% event.eventData contains your requested data type from a dataset.
+% use \r\n for a new line.
+% tip: You can write multiple lines by using:
+%     string = ['My long strings first line\r\n', ...
+%               'The second line!', ...
+%               'Still the second line!\r\nThe Third line!'];
+% if out == '', no load function will be written.
+% Any change to event will be saved for the runFunction
+out =   [...
+    'event.feedbackCor      = imread(C:\image); \r\n'...
+    'event.feedbackInc      = imread(C:\image); \r\n'...
+    'event.feedbackMis      = imread(C:\image); \r\n'...
+        ];
+end
+
+function out = getRunFunction()
+%event.eventData contains your requested data type from a dataset.
+%windowPtr contains the Psychtoolbox window pointer (handle)
+%reply is the struct in which you can create fields to save data
+%reply.timeEventStart contains the time passed since the start of the event
+%startTime contains the time since the start of the block (excl. loading)
+% To change the flow of the events (eg: go 2 events back)
+% you can use variable: eventIter
+% Also nEvents variable might come in handy
+% use \r\n for a new line.
+% tip: You can write multiple lines by using:
+%     string = ['My long strings first line\r\n', ...
+%               'The second line!', ...
+%               'Still the second line!\r\nThe Third line!'];
+out =   [...
+    'event.index; \r\n'...
+    'replyData{eventIter-event.index};                          \r\n'...
+    'cue        = replyData{eventIter-event.index(1)}.image;    \r\n'...
+    'audTarget  = replyData{eventIter-event.index(2)}.audio;    \r\n'...
+    'visTarget  = replyData{eventIter-event.index(3)}.image;    \r\n'...
+    'key        = replyData{eventIter-event.index(4)}.key;      \r\n'...
+    '                                                           \r\n'... 
+    'if  (strcmp(cue,''Small_Fixation.JPG''); reply.cue = "V";         \r\n'...
+    'elseif  (strcmp(cue,''Small_Fixation.JPG''); reply.cue = "A";   \r\n'... 
+    'end                                                                    \r\n'... 
+    'if      strcmp(visTarget,''L.bmp''); reply.visTar = ''L'';             \r\n'... 
+    'elseif  strcmp(visTarget,''C.bmp''); reply.visTar = ''C'';             \r\n'... 
+    'elseif  strcmp(visTarget,''R.bmp''); reply.visTar = ''R'';             \r\n'... 
+    'end  \r\n'... 
+    'if     strcmp(audTarget,''500'');      reply.audTar = ''500'';         \r\n'... 
+    'elseif strcmp(audTarget,''1000.wav''); reply.audTar = ''1000'';        \r\n'... 
+    'elseif strcmp(audTarget,''2000.wav''); reply.audTar = ''2000'';        \r\n'... 
+    '  \r\n'... 
+    'if strcmp(reply.cue,''V'') && strcmp(reply.visTar, ''L'')    && strcmp(reply.key, ''left'')   ||'... 
```

Comment:
Can be replaced with:
`if sum(strcmp({reply.cue, reply.visTar, reply.key}, {'V', 'L', 'left'}))==3 || ....`
Or with
`if sum(strcmp([reply.cue, reply.visTar, reply.key], {['V', 'L', 'left'], ['V', 'C', 'down'], etc}) > 1`
de laatste is gelijk aan de gehele rits aan AND en OR. 
Door `['V', 'L', 'left']` krijg je `'VLleft'` welke je dan compared met de concatinated versie van reply.
Als er in ieder geval 1 waar is (`sum(strcmp(...))>1`) heb je de OR statement met alle ANDs
Volgens mij is dit sneller en korter :)
:+1: